bool call
---
int16[] shape
int32[] semantic_grid